### ISSUE TITLE
Loss goes nan when training dual swin-base

### ISSUE THREAD

**Issue by sean-zhuh** [VDIGPKU/CBNetV2#39]
Hi, I've transferred your code to my own codebase, since your modification of the original mmdetection lies in 3 files (please correct me if I'm wrong): 

1.  CBNetV2/mmdet/models/backbones/cbnet.py
2.  CBNetV2/mmdet/models/necks/cbnet_fpn.py
3.  CBNetV2/mmdet/models/detectors/two_stage.py

I directly do a copy-paste to transfer your code my own version of mmdetection, however, loss goes to nan since epoch 17, the reason is that gradient is overflowing, amp loss scaler has to shrink to a small number, until divided by zero, thus it goes to nan. 

I use default setting of AMP as yours, I can't figure out what's wrong, could you help me?

**Comment by fuweifu-vtoo** [comment]
I meet the same situation! 
have you solved it?

**Comment by sean-zhuh** [comment]
Nope, I've tried to transfer CBNetV2 to MMdetection v2.12.0 but still goes
NaN exactly at 17th epoch.
